sal_interfaces/DIMM/DIMM_Telemetry.xml

```diff
@@ -12,6 +12,64 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>DIMM</Subsystem>
+    <EFDB_Topic>DIMM_sky</EFDB_Topic>
+    <item>
+      <EFDB_Name>status</EFDB_Name>
+      <Description>The current sky status.</Description>
+      <Enumeration>clear=0,lightcoverage=1,cloudy=2,rain=3</Enumeration>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>DIMM</Subsystem>
+    <EFDB_Topic>DIMM_meterology</EFDB_Topic>
+    <item>
+      <EFDB_Name>ambientTemp</EFDB_Name>
+      <Description>The ambient temperature.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>deg_C</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>dewTemp</EFDB_Name>
```

Comment:
dewpoint or dewpointTemperature would be better

Reply:
ok